src/ucs/sys/math.h

```diff
@@ -56,9 +56,10 @@
 
 #define ucs_roundup_pow2(n) \
     ({ \
-        typeof(n) pow2; \
-        ucs_assert((n) >= 1); \
-        for (pow2 = 1; pow2 < (n); pow2 <<= 1); \
+        typeof(n) pow2 = 0; \
+        if ((n) >= 1) { \
+            for (pow2 = 1; pow2 < (n); pow2 <<= 1); \
+        } \
```

Comment:
Please avoid mixes of new features and bugfixes. Clearly something like this can go as a separate PR.